examples/tutorials/lighting_tutorial.py

```diff
@@ -0,0 +1,226 @@
+# [setup]
+import math
+import os
+
+import magnum as mn
+import numpy as np
+from matplotlib import pyplot as plt
+
+import habitat_sim
+from habitat_sim.gfx import LightInfo, LightPositionModel
+from habitat_sim.utils.common import quat_from_angle_axis, quat_to_magnum
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_path = os.path.join(dir_path, "../../data")
+output_path = os.path.join(dir_path, "lighting_tutorial_output/")
+
+save_index = 0
+
+
+def show_img(data, save):
+    plt.figure(figsize=(12, 12))
+    plt.imshow(data, interpolation="nearest")
+    plt.axis("off")
+    plt.show(block=False)
+    if save:
+        global save_index
+        plt.savefig(
+            output_path + str(save_index) + ".jpg", bbox_inches="tight", quality=50
+        )
+        save_index += 1
+    plt.pause(1)
+
+
+def get_obs(sim, show, save):
+    obs = sim.get_sensor_observations()["rgba_camera"]
+    if show:
+        show_img(obs, save)
+    return obs
+
+
+def remove_all_objects(sim):
+    for id in sim.get_existing_object_ids():
+        sim.remove_object(id)
+
+
+def place_agent(sim):
+    # place our agent in the scene
+    agent_state = habitat_sim.AgentState()
+    agent_state.position = [5.0, 0.0, 1.0]
+    agent_state.rotation = quat_from_angle_axis(
+        math.radians(70), y_axis
+    ) * quat_from_angle_axis(math.radians(-20), x_axis)
```

Comment:
This is most probably not possible to solve in this PR alone, but -- I remember we wanted to get rid of the `quaternion` package and those utils and replace them with something else (`mn.Quaternion` or 3x3 matrices?), but it kinda got forgotten since. Here it makes the example look kinda bad because there's two incompatible sets of APIs used for specifying rotations and one has to guess which one is used where.

Reply:
Yeah, not ideal. We should think about unifying the python api.